src/components/Modal/Modal.tsx

```diff
@@ -71,7 +71,7 @@ class ModalClassComponent extends React.Component<Props & NavigationHookProp, St
       <RNModal transparent animationType="fade" visible={visible && isFocused} onRequestClose={onRequestClose}>
         <View style={styles.backdrop}>
           <View style={[styles.container, noPadding && styles.noPadding, !!title && styles.withTitle]}>
-            {title && <Text style={styles.title}>{title}</Text>}
+            {title != null && <Text style={styles.title}>{title}</Text>}
```

Comment:
why display `""`?

Reply:
Yap, let's make it consistent with the styles check.
```suggestion
            {!!title && <Text style={styles.title}>{title}</Text>}
```